vmdb/app/models/miq_provision_task_configured_system_foreman/state_machine.rb

```diff
@@ -61,10 +61,16 @@ def poll_os_built
       requeue_phase
     else
       EmsRefresh.queue_refresh(source)
-      signal :mark_as_completed
+      signal :pre_completion
```

Comment:
The state name `post_provision` would be more consistent with what we use in other state machines.